2020-11-18 17:43:26,310 [INFO] from play.core.server.NettyServer in pool-6-thread-3 - Listening for HTTP on /0:0:0:0:0:0:0:0:9000
2020-11-18 17:43:57,569 [ERROR] from application in New I/O worker #1 - 

! @7hnpfel91 - Internal server error, for (GET) [/] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[cannot find symbol
  symbol:   variable projID
  location: class db.DataAccess]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.sbt.run.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.sbt.run.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader$$anonfun$reload$1.apply(Reloader.scala:323) ~[na:na]
	at play.runsupport.Reloader$$anon$3.run(Reloader.scala:43) ~[na:na]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_112]
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:39) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:321) ~[na:na]
	at play.core.server.DevServerStart$$anonfun$mainDev$1$$anon$1$$anonfun$get$1.apply(DevServerStart.scala:113) ~[play-server_2.11-2.4.8.jar:2.4.8]
	at play.core.server.DevServerStart$$anonfun$mainDev$1$$anon$1$$anonfun$get$1.apply(DevServerStart.scala:111) ~[play-server_2.11-2.4.8.jar:2.4.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.7.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.7.jar:na]
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[na:1.8.0_112]
2020-11-18 17:44:00,511 [ERROR] from application in New I/O worker #2 - 

! @7hnpfepoa - Internal server error, for (GET) [/] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[cannot find symbol
  symbol:   variable projID
  location: class db.DataAccess]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.sbt.run.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.sbt.run.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader$$anonfun$reload$1.apply(Reloader.scala:323) ~[na:na]
	at play.runsupport.Reloader$$anon$3.run(Reloader.scala:43) ~[na:na]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_112]
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:39) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:321) ~[na:na]
	at play.core.server.DevServerStart$$anonfun$mainDev$1$$anon$1$$anonfun$get$1.apply(DevServerStart.scala:113) ~[play-server_2.11-2.4.8.jar:2.4.8]
	at play.core.server.DevServerStart$$anonfun$mainDev$1$$anon$1$$anonfun$get$1.apply(DevServerStart.scala:111) ~[play-server_2.11-2.4.8.jar:2.4.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.7.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.7.jar:na]
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[na:1.8.0_112]
2020-11-18 17:44:48,064 [ERROR] from application in New I/O worker #2 - 

! @7hnpfhi78 - Internal server error, for (GET) [/] ->
 
play.sbt.PlayExceptions$CompilationException: Compilation error[unreported exception java.sql.SQLException; must be caught or declared to be thrown]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.sbt.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.sbt.run.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.sbt.run.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.7.jar:na]
	at play.sbt.run.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.sbt.run.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.sbt.run.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader$$anonfun$reload$1.apply(Reloader.scala:323) ~[na:na]
	at play.runsupport.Reloader$$anon$3.run(Reloader.scala:43) ~[na:na]
	at java.security.AccessController.doPrivileged(Native Method) ~[na:1.8.0_112]
	at play.runsupport.Reloader$.play$runsupport$Reloader$$withReloaderContextClassLoader(Reloader.scala:39) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:321) ~[na:na]
	at play.core.server.DevServerStart$$anonfun$mainDev$1$$anon$1$$anonfun$get$1.apply(DevServerStart.scala:113) ~[play-server_2.11-2.4.8.jar:2.4.8]
	at play.core.server.DevServerStart$$anonfun$mainDev$1$$anon$1$$anonfun$get$1.apply(DevServerStart.scala:111) ~[play-server_2.11-2.4.8.jar:2.4.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.7.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.7.jar:na]
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692) ~[na:1.8.0_112]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[na:1.8.0_112]
2020-11-18 17:46:19,485 [WARN] from application in ForkJoinPool-1-worker-0 - Logger configuration in conf files is deprecated and has no effect. Use a logback configuration file instead.
2020-11-18 17:46:20,486 [WARN] from application in ForkJoinPool-1-worker-0 - application.conf @ file:/Users/frankmeng/Documents/Projects/nlp-workspace/ie-validation/target/scala-2.11/classes/application.conf: 56: user is deprecated, use username instead
2020-11-18 17:46:20,487 [WARN] from application in ForkJoinPool-1-worker-0 - application.conf @ file:/Users/frankmeng/Documents/Projects/nlp-workspace/ie-validation/target/scala-2.11/classes/application.conf: 63: user is deprecated, use username instead
2020-11-18 17:46:20,664 [INFO] from application in ForkJoinPool-1-worker-0 - Creating Pool for datasource 'default'
2020-11-18 17:46:21,129 [WARN] from com.zaxxer.hikari.util.DriverDataSource in ForkJoinPool-1-worker-0 - Registered driver with driverClassName=com.mysql.jdbc.Driver was not found, trying direct instantiation.
2020-11-18 17:46:22,015 [INFO] from play.api.db.DefaultDBApi in ForkJoinPool-1-worker-0 - Database [default] connected at jdbc:mysql://localhost/eies_test
2020-11-18 17:46:22,015 [INFO] from application in ForkJoinPool-1-worker-0 - Creating Pool for datasource 'doc'
2020-11-18 17:46:22,016 [WARN] from com.zaxxer.hikari.util.DriverDataSource in ForkJoinPool-1-worker-0 - Registered driver with driverClassName=com.mysql.jdbc.Driver was not found, trying direct instantiation.
2020-11-18 17:46:22,032 [INFO] from play.api.db.DefaultDBApi in ForkJoinPool-1-worker-0 - Database [doc] connected at jdbc:mysql://localhost/eies_test
2020-11-18 17:46:22,922 [INFO] from play.api.Play in ForkJoinPool-1-worker-0 - Application started (Dev)
2020-11-18 17:46:31,463 [WARN] from application in application-akka.actor.default-dispatcher-2 - application.conf @ file:/Users/frankmeng/Documents/Projects/nlp-workspace/ie-validation/target/scala-2.11/classes/application.conf: 11: application.secret is deprecated, use play.crypto.secret instead
